tools/build_tar/buildtar.go

```diff
@@ -77,6 +80,8 @@ func main() {
 	flag.StringVar(&ownerName, "owner_name", "", "Specify the owner name of all files, e.g. root.root.")
 	flag.Var(&ownerNames, "owner_names", "Specify the owner names of individual files, e.g. path/to/file=root.root.")
 
+	flag.Var(&mtime, "mtime", "Set mtime on tar file entries. May be an integer or the value 'portable' to get the value 2000-01-01, which is usable with non *nix OSes")
```

Comment:
this should be in a separate commit (and probably a separate PR)

Reply:
Done